.github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml

```diff
@@ -0,0 +1,45 @@
+name: Chatbot inference on llama-2-7b-chat-hf
+
+on:
+  workflow_call:
+
+# If there is a new commit, the previous jobs will be canceled
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  HTTP_PROXY_IMAGE_BUILD: http://child-prc.intel.com:913
+  HTTPS_PROXY_IMAGE_BUILD: http://child-prc.intel.com:913
+  HTTP_PROXY_CONTAINER_RUN: http://proxy-chain.intel.com:911
+  HTTPS_PROXY_CONTAINER_RUN: http://proxy-chain.intel.com:911
+
+jobs:
+  inference:
+    name: inference test
+    runs-on: lms-lab
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build Docker Image
+        run: docker build ./ --target cpu --build-arg http_proxy="$HTTP_PROXY_IMAGE_BUILD" --build-arg https_proxy="$HTTPS_PROXY_IMAGE_BUILD" -f workflows/chatbot/inference/docker/Dockerfile -t chatbotinfer:latest && yes | docker container prune && yes | docker image prune
+
+      - name: Start Docker Container
+        run: |
+          cid=$(docker ps -q --filter "name=chatbotinfer")
+          if [[ ! -z "$cid" ]]; then docker stop $cid && docker rm $cid; fi
+          docker run -tid -v /mnt/DP_disk1/huggingface/cache/:/root/.cache/huggingface/hub -v .:/root/chatbot -e http_proxy="$HTTP_PROXY_CONTAINER_RUN" -e https_proxy="$HTTPS_PROXY_CONTAINER_RUN" --name="chatbotinfer" --hostname="chatbotinfer-container" chatbotinfer:latest
+
+      - name: Run Inference Test
+        run: |
+          docker exec "chatbotinfer" bash -c "cd /root/chatbot && source activate && conda activate chatbot-demo; python workflows/chatbot/inference/generate.py --base_model_path \"meta-llama/Llama-2-7b-chat-hf\" --hg_access_token \"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\" --instructions \"Transform the following sentence into one that shows contrast. The tree is rotten.\" "
```

Comment:
We need a secret variable to store the token.

Reply:
> We need a secret variable to store the token.

yes, I originally put HTTP_PROXY related variables under "secrets and variables". However, I cannot access them since PR is from forked repo. You can check with @XuehaoSun . 

Please advice a new place to store the token.

Reply:
moved proxy envs and token to runner's .env.